open_event/templates/admin/model/create_event.html

```diff
@@ -24,18 +24,60 @@
                       {% endcall %}
                   </div>
                   <div class="col-md-6">
-                      <div id="map" style="width:400px; height:300px"></div>
+                   <div id="floating-panel">
+                   <input id="address" type="textbox" value="Sydney, NSW">
+                   <input id="submit" type="button" value="Geocode" onclick="saveAddress()">
+    </div>
+			<div id="map" style="width:400px; height:300px"></div>
                   </div>
               </div>
         </div>
     </div>
     <div class="col-md-2"></div>
     <!-- bring in the google maps library -->
-        <script src="http://www.openlayers.org/api/OpenLayers.js"></script>
-        <script src="{{ url_for('static', filename='admin/js/map.js' )}}"></script>
+     <script>
+function initMap() {
+  var map = new google.maps.Map(document.getElementById('map'), {
+    zoom: 8,
+    center: {lat: -34.397, lng: 150.644}
+  });
```

Comment:
Please add this to the next line so that you drag the initial maker and update the lat/lng before a geocode search, which currently doesn't work:

```
google.maps.event.addListener(init_marker, 'dragend', function (event) { 
$("#latitude").val = this.getPosition().lat(); 
$("#longitude").val = this.getPosition().lng(); 
});
```

Also please start using jquery syntax like above for new code.